Comp2024/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -239,7 +247,8 @@ public void initialize( )
 
   private double getTalonFXDegrees( )
   {
-    return Conversions.rotationsToOutputDegrees(m_rotaryPosition.refresh( ).getValue( ), kRotaryGearRatio);
+    m_rotaryPosition = m_rotaryMotor.getRotorPosition( ).refresh( );
```

Comment:
I don't think this change is doing anything. The status signal we defined should make it update all of the time, and this worked on the wrist.